File: backend/src/main/java/com/tomy/artifyme/artwork/ArtworkRequest.java
package com.tomy.artifyme.artwork;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ArtworkRequest {
    private String userEmail;
    private String sketchedImage;
    private String aiImage;
    private String title;
    private String description;
    private List<PathData> paths;
}

File: backend/Models/DTOs/ArtworkRequest.cs
namespace ArtifyMe.Models.DTOs;

    public class ArtworkRequest
    {
        public string? UserEmail { get; set; }
        public string? SketchedImage { get; set; }
        public string? AiImage { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public List<PathData>? Paths { get; set; }
    }
